test/contracts/timelocks/timelocks.c

```diff
@@ -0,0 +1,26 @@
+#include "../elrond/context.h"
+
+byte counterKey[] = "counter";
+
+void init() {
+    int64storageStore(counterKey, 0);
+}
+
+void incrementCounter() {
+    i64 counter = int64storageLoad(counterKey);
```

Comment:
Storage-related API functions have changed their signature to also include a `keyLength` parameter.

Reply:
done

Reply:
Forgot to push?